src/server/handlers/github.js

```diff
@@ -100,9 +101,8 @@ export const getSnapName = (repositoryUrl, token) => {
         .then((snapcraftYaml) => {
           const snapName = snapcraftYaml.name;
           if (snapName) {
-            return getMemcached().set(cacheId, snapName, 3600, () => {
-              return resolve(snapName);
-            });
+            getMemcached().set(cacheId, snapName, 3600);
+            return resolve(snapName);
```

Comment:
This was causing no snapname to be returned even if one existed when the application is not run with Memcached. There is no reason that the absence of Memcached should cause this feature to fail.